scripts/freesound_download_resample/freesound_resample.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import glob
+import os
+import time
+
+import librosa
+import sox
+from joblib import Parallel, delayed
+
+
+def resample_file(resampled_dir, filepath, ext):
```

Comment:
Add docstring describing what transformation the input will go through - specifically resampling to 16kHz and monochannel + removal of incompatible files.